.lighthouserc.js

```diff
@@ -22,26 +22,31 @@ module.exports = {
       'numberOfRuns': 3,
       'puppeteerScript': 'puppeteer-login-script.js',
       'url': [
-        'http://127.0.0.1:8181/signup?return_url=%2F',
-        'http://127.0.0.1:8181/',
         'http://127.0.0.1:8181/admin',
+         process.env.exploration_editor,
+         process.env.collection_editor,
+         process.env.topic_editor,
+         process.env.skill_editor,
```

Comment:
Can you output just the IDs below and list the interpolated URLs here instead, e.g. [here](https://github.com/GoogleChrome/lighthouse-ci/issues/427#issuecomment-679310842). That way, it's clear what the base URLs are here.

Reply:
Done

Reply:
Done